server/app/jobs/dj_jobs/run_simulate_data_point.rb

```diff
@@ -185,14 +185,22 @@ def perform
       @sim_logger.info "Directory is #{simulation_dir}"
       run_log_file = File.join(run_dir, 'run.log')
       @sim_logger.info "Opening run.log file '#{run_log_file}'"
-
+      #add check for valid CLI option or ""
+	    unless ["","--debug"].include?(@data_point.analysis.cli_debug)
```

Comment:
looks good, can you make sure to move this indent back inline?

Reply:
yep, hidden tab in there. removed it